config/hack.config.ts

```diff
@@ -0,0 +1,189 @@
+    name: 'Shreya Mekala',
+    position: 'Logistics Coordinator',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Roshni Reddhannan',
+    position: 'Logistics Coordinator',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Sahishnu Sagiraju',
+    position: 'Logistics Coordinator',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Alexander Tifase',
+    position: 'Logistics Coordinator',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Bhuvana Madiraju',
+    position: 'Industry Lead',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Sharun Naicker',
+    position: 'Industry Coordinator',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Sachi Hansalia',
+    position: 'Industry Coordinator',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Ridwan Amin',
+    position: 'Industry Coordinator',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Bryan Caraman',
+    position: 'Industry Coordinator',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'John Luke',
+    position: 'Industry Coordinator',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Mike Nguyen',
+    position: 'Tech Lead',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Wei Seng',
+    position: 'Developer',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Rayyan Waris',
+    position: 'Developer',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Victoria Vynnychok',
+    position: 'Developer',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Anh Tu Do',
+    position: 'Developer',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Kelly Zhou',
+    position: 'Maketing Lead',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Mitchell Vu',
+    position: 'Video Editor',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Shaz Kumar',
+    position: 'Social Media',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Jordan Tan',
+    position: 'Designer',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Alyssa To',
+    position: 'Designer/UI/UX',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Ann Jayan',
+    position: 'Designer',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Sarah Jacob',
+    position: 'Designer',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Caleb Bae',
+    position: 'Outreach',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Andrew Goodland',
+    position: 'Finance',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Ayusha Timalsena',
+    position: 'Finance',
+    image: '/assets/officer/OfficerImage.png',
+  },
+  {
+    name: 'Michael :D',
```

Comment:
I think it might be a better idea to change this to `Michael Zhao`.

Reply:
Lol, thought it was him but wasn't 100% sure